tests/Aspire.Hosting.Tests/Backchannel/Exec/ExecTestsBase.cs

```diff
@@ -43,7 +43,37 @@ internal async Task<List<CommandOutput>> ExecWithLogCollectionAsync(
         return logs;
     }
 
-    internal static void AssertLogsContain(List<CommandOutput> logs, params string[] expectedLogMessages)
+    protected async Task<List<ContainerExecCommandOutput>> ProcessAndCollectLogs(IAsyncEnumerable<ContainerExecCommandOutput> containerExecLogs)
+    {
+        var logs = new List<ContainerExecCommandOutput>();
```

Comment:
How will this work with long running operations with lots of log entries.

Reply:
this exists only for tests and to easily check the collected logs as collection